src/assets/styles/styles/custom/_error-list-item.scss

```diff
@@ -0,0 +1,138 @@
+.error-list-item {
+  border-radius: 2px;
+  border-style: solid;
+  border-width: 1px;
+  border-color: $error-list-item-border-color;
+  margin-left: 24px;
+  margin-right: 23px;
+  margin-bottom: 8px;
+  display: flex;
+  align-items: stretch;
+  position: relative;
+
+  .red-rectangle {
+    border-radius: 4px;
+    width: 0.8%;
+    position: absolute;
+    height: 100%;
+    background-color: $error-red;
+  }
+
+  .error-title {
+    color: $error-date-text-color;
+    size: 14px;
+    width: 20%;
+    text-align: center;
+    padding-left: 20px;
+    position: relative;
+
+    .error-title-content {
+      font-family: "Roboto";
+      position: absolute;
+      top: 50%;
+      transform: translate(0, -50%);
+      padding-left: 10px;
+
+      @media (max-width: 1150px) {
+        display: block;
+        width: 100%;
+        transform: none;
+        bottom: auto;
+        left: auto;
+        position: unset;
+        text-align: left;
+        padding-top: 10px;
+      }
+    }
+
+    @media (max-width: 1700px) {
+      width: 25%;
+      padding-left: 10px;
+    }
+
+    @media (max-width: 1350px) {
+      width: 30%;
+    }
+
+    @media (max-width: 1150px) {
+      display: block;
+      width: 100%;
+      position: unset;
+    }
+  }
+
+  .error-text {
+    color: $primary-text-color;
+    size: 13px;
+    width: 62.2%;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    text-align: justify;
+    font-family: "Roboto";
+
+    @media (max-width: 1700px) {
+      width: 55%;
+    }
+
+    @media (max-width: 1500px) {
+      width: 50%;
+    }
+
+    @media (max-width: 1350px) {
+      width: 43%;
+    }
+
+    @media (max-width: 1150px) {
+      display: block;
+      width: 100%;
+      position: unset;
+      padding-left: 20px;
+      padding-right: 10px;
+    }
+  }
+
+  .error-btn {
+    position: relative;
+    height: 100%;
+    width: 17%;
+
+    Button {
+      position: absolute;
+      bottom: 50%;
+      left: 50%;
+      transform: translate(-50%, 50%);
+
+      @media (max-width: 1150px) {
+        display: block;
+        transform: none;
+        bottom: auto;
+        left: auto;
+        position: unset;
+      }
+    }
+
+    @media (max-width: 1700px) {
+      width: 20%;
+    }
+
+    @media (max-width: 1500px) {
+      width: 25%;
+    }
+
+    @media (max-width: 1350px) {
+      width: 27%;
+    }
+
+    @media (max-width: 1150px) {
+      display: block;
+      width: 100%;
+      position: unset;
+      padding-left: 10px;
+      padding-bottom: 10px;
+    }
+  }
+
+  @media (max-width: 1150px) {
+    display: block;
```

Comment:
Przy skalowaniu elementy zmieniają układ na blokowy:
![skalowanie](https://user-images.githubusercontent.com/48785655/101985427-3c4f9780-3c88-11eb-89ea-501aa94d88d8.png)
To mój wymysł, na Figmie nie było przypadku dla zmiejszenia okna.

Reply:
Moim zdaniem, świetny pomysł :D